components/atoms/ResourceCard.js

```diff
@@ -0,0 +1,43 @@
+import Image from "next/image";
+import Link from "next/link";
+import PropTypes from "prop-types";
+
+/**
+ *  Available Resources Card Component
+ */
+export default function ResourceCard(props) {
+  return (
+    <div className="md:shadow-lg h-20 w-auto rounded-md border-2 pl-2 cursor-pointer hover:bg-gray-light-report focus:ring-1 focus:ring-black focus:ring-offset-1 active:bg-gray-active">
+      <Link href="/">
+        <div className="grid grid-cols-2 grid-flow-row">
```

Comment:
We still need to include the `<a>` tag here (after `Link`, I think), otherwise we can't focus on them with the keyboard. Note that it might turn all your text blue.

Reply:
i see... k ill look at options